src/beneath-std.md

```diff
@@ -39,11 +42,12 @@ in the same format as C (aside from the exact integer types being used):
 #![allow(internal_features)]
 #![no_std]
 
-// Necessary for `panic = "unwind"` builds on some platforms.
+// Necessary for `panic = "unwind"` builds on cfg(unix) platforms.
 #![feature(panic_unwind)]
 extern crate unwind;
 
 // Pull in the system libc library for what crt0.o likely requires.
+#[cfg(not(windows))]
```

Comment:
I suspect windows-gnu still needs it, but I'm not entirely sure.

Reply:
windows-gnu includes gnu libc [via the compiler](https://github.com/rust-lang/rust/blob/6579ed89f0fcc26da71afdd11d30d63f6f812a0a/compiler/rustc_target/src/spec/base/windows_gnu.rs#L30). The libc crate doesn't add anything, as far as I'm aware.